mrp_subcontracting_skip_no_negative/tests/test_mrp_subcontracting_skip_no_negative.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo.exceptions import ValidationError
+from odoo.tests import Form
+
+from odoo.addons.mrp_subcontracting.tests.common import TestMrpSubcontractingCommon
+
+
+class TestMrpSubcontractingSkipNoNegative(TestMrpSubcontractingCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(
+            context=dict(
+                cls.env.context,
+                test_stock_no_negative=True,
+            )
+        )
+
+    def test_mrp_subcontracting_skip_no_negative(self):
+        picking_form = Form(self.env["stock.picking"])
+        picking_form.picking_type_id = self.env.ref("stock.picking_type_in")
+        picking_form.partner_id = self.subcontractor_partner1
+        with picking_form.move_ids_without_package.new() as move:
+            move.product_id = self.finished
+            move.product_uom_qty = 1
+        subcontracting_receipt = picking_form.save()
```

Comment:
What I mean is this picking is a manual one, not the PO generated one (with the MO linked to it).

Reply:
Creating a subcontracting with an in picking also generates the subcontracting flow (without the need for PO + confirm). This same approach is used in the tests of many modules, including `mrp_subcontracting`